lib/ansible/modules/packaging/os/rpm_key.py

```diff
@@ -143,12 +143,9 @@ def fetch_key(self, url):
     def normalize_keyid(self, keyid):
         """Ensure a keyid doesn't have a leading 0x, has leading or trailing whitespace, and make sure is uppercase"""
         ret = keyid.strip().upper()
-        if ret.startswith('0x'):
+        if ret.startswith('0x') or ret.startswith('0X'):
             return ret[2:]
-        elif ret.startswith('0X'):
-            return ret[2:]
-        else:
-            return ret
+        return ret
```

Comment:
The previous line sets ret to an uppercase version of the string so we only need to check with the uppercase version in this conditional:
``` python
ret = keyid.strip().upper()
if ret.startswith('0X'):
    return ret[2:]
return ret
```